apps/backend/src/db/migrations/20240823123910_create_onboarding_answers_table.ts

```diff
@@ -0,0 +1,41 @@
+import { type Knex } from "knex";
+
+const TableName = {
+	ONBOARDING_ANSWERS: "onboarding_answers",
+	ONBOARDING_QUESTIONS: "onboarding_questions",
+} as const;
+
+const ColumnName = {
+	CREATED_AT: "created_at",
+	ID: "id",
+	LABEL: "label",
+	QUESTION_ID: "question_id",
+	UPDATED_AT: "updated_at",
+} as const;
+
+async function up(knex: Knex): Promise<void> {
+	await knex.schema.createTable(TableName.ONBOARDING_ANSWERS, (table) => {
+		table.increments(ColumnName.ID).primary();
+		table.string(ColumnName.LABEL).unique().notNullable();
+		table
+			.integer(ColumnName.QUESTION_ID)
+			.references(ColumnName.ID)
+			.inTable(TableName.ONBOARDING_QUESTIONS)
+			.onDelete("CASCADE")
```

Comment:
```suggestion
			.onDelete(DELETE_STRATEGY)
```
Move to a separate variable